test/Library-test.js

```diff
@@ -106,24 +112,103 @@ describe("library.js", function() {
       var result1 = checkoutBook(denverLibrary, "Pride and Prejudice", "fiction");
 
       assert.deepEqual(denverLibrary.shelves, {fantasy: [dracula], fiction: [], nonFiction: [bornACrime]});;
-      assert.equal(result1, "You have now checked out Pride and Prejudice from the Denver Public Library");
+      assert.equal(result1, "You have now checked out Pride and Prejudice from the Denver Public Library.");
 
       var result2 = checkoutBook(denverLibrary, "Born a Crime", "nonFiction");
 
       assert.deepEqual(denverLibrary.shelves, {fantasy: [dracula], fiction: [], nonFiction: []});;
-      assert.equal(result2, "You have now checked out Born a Crime from the Denver Public Library")
+      assert.equal(result2, "You have now checked out Born a Crime from the Denver Public Library.")
     });
 
     it.skip("should only checkout a book if the book is on the shelves", function() {
       var denverLibrary = createLibrary("Denver Public Library");
 
       var error1 = checkoutBook(denverLibrary, "The Fifth Season", "fantasy");
 
-      assert.equal(error1, "Sorry, there are currently no copies of The Fifth Season available at the Denver Public Library");
+      assert.equal(error1, "Sorry, there are currently no copies of The Fifth Season available at the Denver Public Library.");
 
       var error2 = checkoutBook(denverLibrary, "Yes Please", "nonFiction");
 
-      assert.equal(error2, "Sorry, there are currently no copies of Yes Please available at the Denver Public Library");
+      assert.equal(error2, "Sorry, there are currently no copies of Yes Please available at the Denver Public Library.");
+    });
+  });
+
+  describe('takeStock', function() {
+
+    it.skip('should be able to take stock of how many books are on a shelf', function() {
+      var hyperion = {
+        title: "Hyperion",
+        mainCharacter: { name: "The Shrike", age: null, pronouns: "they" },
+        pageCount: 482,
+        genre: "fantasy"
+      };
+
+      var dune = {
+        title: "Dune",
+        mainCharacter: { name: "Paul", age: 15, pronouns: "he/him" },
+        pageCount: 421,
+        genre: "fantasy"
+      };
+
+      var prideAndPrejudice = {
+        title: "Pride and Prejudice",
+        mainCharacter: { name: "Eliabeth Bennet", age: 20, pronouns: "she/her" },
+        pageCount: 432,
+        genre: "fiction"
+      }
+
+      var auroraLibrary = createLibrary("Aurora Public Library");
+
+      addBook(auroraLibrary, hyperion);
+      addBook(auroraLibrary, dune);
+      addBook(auroraLibrary, prideAndPrejudice);
+
+      assert.equal(takeStock(auroraLibrary, 'fantasy'), "There are a total of 2 fantasy books at the Aurora Public Library.")
```

Comment:
Love the test for checking a type of book has been added! And then having a following test for checking a different shelf just to make sure nothing has been hardcoded is terrific!